Sources/Visualization/Microsoft.Psi.Visualization.Windows/ViewModels/DatasetViewModel.cs

```diff
@@ -428,9 +428,10 @@ public AuxiliaryStreamInfo ShowAuxiliaryStreamInfo
         /// Loads a dataset from the specified file.
         /// </summary>
         /// <param name="filename">The name of the file that contains the dataset to be loaded.</param>
+        /// <param name="autoSave">A value to indicate whether to enable the autosave feature.</param>
         /// <returns>The newly loaded dataset view model.</returns>
-        public static DatasetViewModel Load(string filename) =>
-            new DatasetViewModel(Dataset.Load(filename))
+        public static DatasetViewModel Load(string filename, bool autoSave) =>
```

Comment:
Should there be a default value of `false` on this, like with `LoadAsync`?

Reply:
Good catch, just updated it